pandas/tseries/tests/test_timeseries.py

```diff
@@ -2249,15 +2245,26 @@ def test_concat_datetime_datetime64_frame(self):
     def test_period_resample(self):
         # GH3609
         s = Series(range(100), index=date_range(
-            '20130101', freq='s', periods=100), dtype='float')
+            '20130101', freq='s', periods=100, name='idx'), dtype='float')
         s[10:30] = np.nan
         expected = Series([34.5, 79.5], index=[Period(
             '2013-01-01 00:00', 'T'), Period('2013-01-01 00:01', 'T')])
+        expected.index.name = 'idx'
         result = s.to_period().resample('T', kind='period').mean()
         assert_series_equal(result, expected)
         result2 = s.resample('T', kind='period').mean()
         assert_series_equal(result2, expected)
 
+    def test_empty_period_index_resample(self):
```

Comment:
This now tests [this](https://github.com/pydata/pandas/pull/12771/files#diff-e21e44fc4ab14efc2f09e0dacdd79fc6R676), which was untested